misc/scripts/install_d3d12_sdk_windows.py

```diff
@@ -81,6 +92,21 @@
 print(f"Extracting WinPixEventRuntime {pix_version} to {pix_folder} ...")
 shutil.unpack_archive(pix_archive, pix_folder, "zip")
 os.remove(pix_archive)
+if has_mingw:
```

Comment:
Maybe print a message if MinGW isn't found, so that the user is aware that the MinGW-friendly files were not generated (and that they should re-run the script if they decide to use MinGW later).